<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_prescan</class>
 <widget class="QDialog" name="window_prescan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prescan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DSXW.qrc">
    <normaloff>:/DSXW/DSXW.png</normaloff>:/DSXW/DSXW.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1">
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>318</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="vtd_vehicle_model_select_2">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
border-radius:10px</string>
       </property>
       <property name="text">
        <string>Prescan</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>358</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_13">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="blank_7">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vtd_vehicle_model_select_3">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
border-radius:10px</string>
       </property>
       <property name="text">
        <string>1.选择输入文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blank_8">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="prescan_select_input">
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:2px solid rgb(85, 87, 83);
background-color: rgb(186, 189, 182);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(85, 87, 83);
background-color: rgb(175, 175, 175);
border-radius:10px;
}
QPushButton:pressed{
border:2px solid rgb(85, 87, 83);
background-color: rgb(92, 105, 107);
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>选择xosc文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prescan_select_input_dir">
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border:2px solid rgb(85, 87, 83);
background-color: rgb(186, 189, 182);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(85, 87, 83);
background-color: rgb(175, 175, 175);
border-radius:10px;
}
QPushButton:pressed{
border:2px solid rgb(85, 87, 83);
background-color: rgb(92, 105, 107);
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>选择文件夹</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="prescan_show_input">
       <property name="minimumSize">
        <size>
         <width>541</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
background-color: rgb(255, 255, 255);
border-radius:10px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="5">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="6">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <spacer name="horizontalSpacer_14">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <spacer name="horizontalSpacer_15">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="blank_9">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vtd_vehicle_model_select_4">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
border-radius:10px</string>
       </property>
       <property name="text">
        <string>2.选择车辆模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blank_10">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="prescan_vehicle_model_select">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:2px solid rgb(186,186,186);
background-color: rgb(250,250,250);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(186,186,186);
background-color: rgb(240, 240, 240);
border-radius:10px;}
QPushButton:pressed{
border:2px solid rgb(186,186,186);
background-color: rgb(211, 215, 207);
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>选择车辆模型</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="prescan_vehicle_model_show">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>541</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
background-color: rgb(255, 255, 255);
border-radius:10px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="3">
    <spacer name="horizontalSpacer_16">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="3">
    <spacer name="horizontalSpacer_17">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="blank_12">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vtd_vehicle_model_select_5">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
border-radius:10px</string>
       </property>
       <property name="text">
        <string>3.选择传感器模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blank_11">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="prescan_sensor_select">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:2px solid rgb(186,186,186);
background-color: rgb(250,250,250);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(186,186,186);
background-color: rgb(240, 240, 240);
border-radius:10px;}
QPushButton:pressed{
border:2px solid rgb(186,186,186);
background-color: rgb(211, 215, 207);
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>选择传感器模型</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="prescan_sensor_show">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>541</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
background-color: rgb(255, 255, 255);
border-radius:10px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="3">
    <spacer name="horizontalSpacer_21">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="3">
    <spacer name="horizontalSpacer_18">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QPushButton" name="blank_13">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="vtd_vehicle_model_select_6">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
border-radius:10px</string>
       </property>
       <property name="text">
        <string>4.选择输出目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="blank_14">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>6</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:5px solid rgb(186,186,186);
border-radius:25px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="prescan_select_output">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:2px solid rgb(85, 87, 83);
background-color: rgb(186, 189, 182);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(85, 87, 83);
background-color: rgb(175, 175, 175);
border-radius:10px;
}
QPushButton:pressed{
border:2px solid rgb(85, 87, 83);
background-color: rgb(92, 105, 107);
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>输出到Prescan工程目录</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="prescan_show_output">
       <property name="minimumSize">
        <size>
         <width>541</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid rgb(186,186,186);
background-color: rgb(255, 255, 255);
border-radius:10px</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="18" column="3">
    <spacer name="horizontalSpacer_23">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="3">
    <spacer name="horizontalSpacer_22">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>548</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="to_prescan">
       <property name="minimumSize">
        <size>
         <width>291</width>
         <height>41</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:2px solid rgb(85, 87, 83);
background-color: rgb(190, 221, 255);
border-radius:10px
}
QPushButton:hover{
border:2px solid rgb(85, 87, 83);
background-color: rgb(170, 200, 230);
border-radius:10px;
}
QPushButton:pressed{
border:2px solid rgb(85, 87, 83);
background-color: rgb(138, 164, 193);
border-radius:10px;
}</string>
       </property>
       <property name="text">
        <string>转换为Prescan格式</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="21" column="3">
    <spacer name="horizontalSpacer_25">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="22" column="3">
    <spacer name="horizontalSpacer_24">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="DSXW.qrc"/>
 </resources>
 <connections/>
</ui>
